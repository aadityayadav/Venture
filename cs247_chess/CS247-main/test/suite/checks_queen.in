setup
+ K a8
+ Q e4
+ k a4
done
- a4
+ k e1
done
- e1
+ k h4
done
- h4
+ k e8
done
- e8
+ k b7
done
- b7
+ k h7
done
- h7
+ k h1
done
- h1
+ k b1
done
- b1
+ k a1
done
