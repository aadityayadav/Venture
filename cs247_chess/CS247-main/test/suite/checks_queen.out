The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
K_______
________
________
________
____Q___
________
________
k_______
